shiny.Rmd

```diff
@@ -1,5 +1,5 @@
 ---
-title: Using DT in Shiny
+title: "Using DT in Shiny"
```

Comment:
An unnecessary [cosmetic change](https://yihui.name/en/2018/11/cosmetic-changes/) but that's fine this time.

Reply:
Strange, I didn't change that.